app/main/service/solarService.py

```diff
@@ -39,6 +39,16 @@ def get_site_energy(self,api_key,startDate,endDate,timeUnit,site_id)->ReponseMod
             return ReponseModel(message="Unauthorized",status=401)
         except:
             return ReponseModel(message=str(requests.exceptions.HTTPError),status=500)
+
```

Comment:
This is the same as the get_site_energy please use the exisiting get_site_energy for get_d_function()